snapcraft.yaml

```diff
@@ -17,24 +18,27 @@ package-repositories:
     ppa: snappy-dev/image
 
 confinement: strict
-grade: stable
+grade: devel
 parts:
   gadget:
     plugin: nil
     source: .
     override-build: |
       set -x
-      # unconditionally set the arch-triplet since this snapcraft.yaml will 
-      # always be used to produce an armhf snap
+      
+      # we only support arm64 for 24 for pi
       BUILD_ARCH_TRIPLET=aarch64-linux-gnu
+      
       # but only use the cross-compiling sources when actually cross compiling
       # as otherwise its just safer to use the build environment - it offers
       # much more flexibility.
       if [ "$SNAPCRAFT_ARCH_TRIPLET" = "x86_64-linux-gnu" ] || [ -z "$SNAPCRAFT_ARCH_TRIPLET" ]; then
         OPTIONAL_ARGS="SERIES_HOST=\"focal\" SOURCES_HOST=\"./helpers/cross-sources.list\" SOURCES_D_HOST=\"./helpers\""
```

Comment:
Still a `focal` here, also I think there should be one variable here (`SERIES=noble`) that is passed down everywhere, and probably use sed to set the series in the cross-sources.list file.

Reply:
good point, I added something similar to this.